### PR TITLE
fix typo in triagebot.toml

### PR DESCRIPTION
**Pull request by naskya**
This pull request fixes a minor typo in the triagebot config file.

### CODE REVIEW

File: triagebot.toml [hunk 1]
```diff
@@ -823,7 +823,7 @@ If appropriate, please update `CONFIG_CHANGE_HISTORY` in `src/bootstrap/src/util
 message = "This PR changes how LLVM is built. Consider updating src/bootstrap/download-ci-llvm-stamp."
 
 [mentions."test/crashes"]
-message = "This PR changes a file inside `tests/crashes`. If a crash was fixed, please move into the correspondig `ui` subdir and add 'Fixes #<issueNr>' to the pr description to autoclose the issue upon merge."
+message = "This PR changes a file inside `tests/crashes`. If a crash was fixed, please move into the corresponding `ui` subdir and add 'Fixes #<issueNr>' to the pr description to autoclose the issue upon merge."
```
Comment (workingjubilee) [inline triagebot.toml:826]:
```suggestion
message = "This PR changes a file inside `tests/crashes`. If a crash was fixed, please move into the corresponding `ui` subdir and add 'Fixes #<issueNr>' to the PR description to autoclose the issue upon merge."
```

**Comment by compiler-errors** [comment]
I would appreciate if you squashed this into a single commit as well!

**Review by jieyouxu — APPROVED** [review]
Thanks!

**Pull request merged**